packages/libiconv.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Libiconv < Package
+  description 'GNU charset conversion library for libc which does not implement it.'
+  homepage 'https://www.gnu.org/software/libiconv/'
+  version '1.15'
+  source_url 'https://mirror.ibcp.fr/pub/gnu/libiconv/libiconv-1.15.tar.gz'
```

Comment:
Please use `https://ftpmirror.gnu.org/libiconv/libiconv-1.15.tar.gz` and not the actual mirror url.